pkg/api/v20220904/openshiftinstallversions_convert.go

```diff
@@ -0,0 +1,18 @@
+package v20220904
+
+import "github.com/Azure/ARO-RP/pkg/api"
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+type installOpenShiftVersionsConverter struct{}
+
+// TODO: Change the comment
```

Comment:
nit: does this todo still apply? if not please remove it

Reply:
Ohh wow .. nice catch .. thank you. Corrected it.